azurerm/helpers/azure/servicebus.go

```diff
@@ -73,6 +87,67 @@ func FlattenServiceBusAuthorizationRuleRights(rights *[]servicebus.AccessRights)
 }
 
 //shared schema
+func MergeSchema(a map[string]*schema.Schema, b map[string]*schema.Schema) map[string]*schema.Schema {
```

Comment:
since this isn't being used outside of this package, this could probably become a private method?